contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -181,6 +181,69 @@ configured. Conformance tests must be able to be run against clusters that have
 not been created with `test-infra/kubetest`, just provide a kubeconfig with the
 appropriate endpoint and credentials.
 
+### Running Conformance Tests With [KinD](https://kind.sigs.k8s.io/)
+
+1. Work in your kubernetes branch, preferably in the default go src location
```

Comment:
let's be explicit here by adding `$GOPATH/src/k8s.io/kubernetes` right after "go src location".

Reply:
updated